client/components/tile-grid/tile.jsx

```diff
@@ -0,0 +1,67 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+
+export default class extends React.PureComponent {
+	static propTypes = {
+		buttonClassName: PropTypes.string,
+		buttonLabel: PropTypes.string,
+		className: PropTypes.string,
+		description: PropTypes.string,
+		href: PropTypes.string,
+		image: PropTypes.string,
+		onClick: PropTypes.func,
+	};
+
+	render() {
+		const {
+			buttonClassName,
+			buttonLabel,
+			className,
+			description,
+			href,
+			image,
+			onClick,
+		} = this.props;
+		const isClickable = href || onClick;
+		const TileElement = isClickable ? 'a' : 'span';
+		const tileClassName = classNames(
+			'tile-grid__item',
+			{
+				'is-clickable': isClickable,
+			},
+			className
+		);
+
+		return (
+			<Card className={ tileClassName }>
+				<TileElement className="tile-grid__item-link" href={ href } onClick={ onClick }>
```

Comment:
Hmm, do we really need a `TileElement` (which is either an `a` or a `span`) inside a `Card`? I thought `Card` itself was pretty much the same thing...

Reply:
Ah, well that's mainly because clickable `Card` items contain a link indicator and some additional CSS:

![](https://cldup.com/hQMPzqyi71.png)

So we'll have to reset that CSS if we want to go that way. I attempted that for some other purpose (I think it was in `Banner`) and it didn't work out very well. I prefer not coupling this too much to the functionality of `Card`, given the circumstances.

Reply:
Hmm, I'm still a bit adamant to relying on `Card`, my argument being that it so closely overlaps in functionality -- accepting `href` and/or `onClick` to become 'clickable'. I think the styling based fix would amount to setting `.card__link-indicator` to `display: none` for `TileGrid`...

But again, your decision ultimately 😄

Reply:
I'm happy to give that a try 😉  Stay tuned.

Reply:
Done in 4e05920b4cfada218998e88318519cebf3e0c752.

Reply:
And I have to admit it's a great improvement, helped us reduce the CSS a bit. Thanks again 👍 💯